src/node/sockets/node-server/API/Node-API-Private.js

```diff
@@ -21,15 +21,15 @@ class NodeAPIPrivate{
 
     async walletImport(req, res){
 
-        let privateKey = req.privateKey;
-
-        let address = InterfaceBlockchainAddressHelper.generateAddress(undefined, privateKey);
```

Comment:
InterfaceBlockchainAddressHelper.generateAddress throws an error that `privateKey` is not a buffer